ponyo/generate_data_parallel.py

```diff
@@ -782,8 +778,9 @@ def apply_correction_io(
 
     Returns
     --------
-    Files of simulated data with different numbers of experiments added.
-    Each file named as "Experiment_<number of experiments added>"
+    Files of simulated data with different numbers of experiments added and corrected for.
+    Each file named as "Experiment_<number of experiments added>".
+    After the data is corrected, the dimensions are now gene x sample
```

Comment:
I don't think this function actually returns the files, just writes them to the disk, right? You may want to clarify that in case the user is expecting a list of filenames to come out of the function

Reply:
Good point. I will change this